### PR TITLE
Scala 2.11 cross-compilation & Slick upgrade to 3.2.0 final

### PR DESCRIPTION
**Pull request by studiodev**
Hello,

This PR aims to use your lib with Slick 3.2.0 with Scala 2.11.x. I've just enable the crossScalaCompilation in your build, and it seems to work (tested with `+ publishLocal`).

A new release with this scala 2.11 compatibility would be highly appreciated :)

### CODE REVIEW

File: src/main/scala/slickless/HListShape.scala [hunk 1]
```diff
@@ -7,7 +7,7 @@ import slick.ast.MappedScalaType
 import slick.lifted.{ Shape, ShapeLevel, FlatShapeLevel, MappedProductShape, MappedProjection }
 
 final class HListShape[L <: ShapeLevel, M <: HList, U <: HList : ClassTag, P <: HList]
-    (val shapes: Seq[Shape[_, _, _, _]]) extends MappedProductShape[L, HList, M, U, P] {
+    (val shapes: Seq[Shape[_<: ShapeLevel, _, _, _]]) extends MappedProductShape[L, HList, M, U, P] {
```
Comment (d6y) [inline src/main/scala/slickless/HListShape.scala]: Was this change required for 2.11, or... something else?

**Comment by d6y** [comment]
Thanks @studiodev - I've left one question on a change, but great! Thank you... we'll get a build out as soon as we can and I'll post an update here.

File: src/main/scala/slickless/HListShape.scala (hunk 1 above)
Comment (studiodev) [inline src/main/scala/slickless/HListShape.scala]: It's related to slick version bump (to 3.2.0), the first commit.
The cross compilation seems transparent

Comment (d6y) [inline src/main/scala/slickless/HListShape.scala]: OK: it was a change in Slick between M2 and final:

```
HListShape.scala:10: overriding value shapes in class ProductNodeShape of type Seq[slick.lifted.Shape[_ <: slick.lifted.ShapeLevel, _, _, _]];
[error]  value shapes has incompatible type
[error]     (val shapes: Seq[Shape[_, _, _, _]]) extends MappedProductShape[L, HList, M, U, P] {
[error]          ^
[error] one error found
```

**Comment by davegurnell** [comment]
This looks great! Thanks!

**Comment by d6y** [comment]
That's being pushed out to Maven central. Give it a couple of hours and it should be there as version 0.3.2

https://github.com/underscoreio/slickless/releases/tag/0.3.2

**Comment by studiodev** [comment]
Perfect, thanks :)
